addon/services/syncer.js

```diff
@@ -217,7 +218,9 @@ export default Ember.Service.extend({
 
       return this.get('offlineStore').query(modelName, builder.build()).then((jobs) => {
         dexieService.set('queueSyncUpTotalWorksCount', jobs.get('length'));
-        return this._runJobs(store, jobs, options && options.continueOnError);
+        return options.useBatchUpdate
+          ? this._runJobsThroughBatch(store, jobs)
```

Comment:
Параметр `options` не обязательный.
```suggestion
        return options && options.useBatchUpdate ?
          this._runJobsThroughBatch(store, jobs)
```